holmes/core/tool_calling_llm.py

```
@@ -376,6 +393,7 @@ def _invoke_tool(
         self,
         tool_to_call: ChatCompletionMessageToolCall,
         previous_tool_calls: list[dict],
+        trace_span=DummySpan(),
```

Ruff (0.12.2)

Do not perform function call `DummySpan` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)